lumigator/python/mzai/backend/services/experiments.py

```diff
@@ -134,7 +134,7 @@ def create_experiment(self, request: ExperimentCreate) -> ExperimentResponse:
 
         runtime_env = {
             "pip": settings.PIP_REQS,
```

Comment:
Sorry I missed this earlier - why don't we just pass the `requirements.txt` file that comes in the lm-buddy (sorry, evaluator) directory? According to [ray docs](https://docs.ray.io/en/latest/ray-core/handling-dependencies.html#api-reference) this should be possible and

- allows ppl to provide requirements together with their jobs
- does not require settings to know the position of lm-buddy's requirements (removing the need to update this if anything's moved or added)

WDYT?

Reply:
this is what we're doing, but passing through settings because I wanted to consolidate all files, variables, etc in one place. I wouldn't want the filepath to be in the code directly since that could change. In the case of the direct filepath, this module would have to know where the filepath is relative to the `services` directory anyway. It's a tricky issue to resolve because most Ray installs assume you're running the script from the same directory the code is.

Reply:
Gotcha! One possible suggestion we discussed: what if we set `working_dir` to the actual evaluator dir and just consider it as some independent package? Eventually rn it has no lumigator deps and when we run it on ray it'll just be its own thing. This allows us to only reference its dir once (in working_dir) and pip install requirements.txt because it'll be directly accessible in the working directory

Reply:
works for now!